src/management-system-v2/app/api/private/[environmentId]/image-helpers.ts

```diff
@@ -0,0 +1,70 @@
+import { fileTypeFromBuffer } from 'file-type';
+import { NextRequest, NextResponse } from 'next/server';
+import stream from 'node:stream';
+import type { ReadableStream } from 'node:stream/web';
+
+export function invalidRequest(request: NextRequest) {
+  const allowedContentTypes = ['image/jpeg', 'image/svg+xml', 'image/png'];
+
+  const contentType = request.headers.get('content-Type');
```

Comment:
Header names are case-insensitive, but we should probably stick to one regardless :)